libs/sdk-ui-filters/src/RankingFilter/RankingFilterDropdownBody.tsx

```diff
@@ -0,0 +1,126 @@
+// (C) 2020 GoodData Corporation
+import React, { useState, useCallback } from "react";
+import Button from "@gooddata/goodstrap/lib/Button/Button";
+import { IRankingFilter, LocalIdRef } from "@gooddata/sdk-model";
+import { WrappedComponentProps, injectIntl, FormattedMessage } from "react-intl";
+import { isEqual, xorWith, isEmpty } from "lodash";
+import { IMeasureDropdownItem, IAttributeDropdownItem } from "./types";
+import { OperatorDropdown } from "./OperatorDropdown/OperatorDropdown";
+import { ValueDropdown } from "./ValueDropdown/ValueDropdown";
+import { AttributeDropdown } from "./AttributeDropdown/AttributeDropdown";
+import { MeasureDropdown } from "./MeasureDropdown/MeasureDropdown";
+
+const isApplyButtonDisabled = (filter: IRankingFilter, filterState: IRankingFilter) => {
+    const rankingFilter = filter.rankingFilter;
+    const rankingFilterState = filterState.rankingFilter;
+
+    const operatorNotChanged = rankingFilter.operator === rankingFilterState.operator;
+    const valueNotChanged = rankingFilter.value === rankingFilterState.value;
+    const attributesNotChanged = isEmpty(
+        xorWith(rankingFilter.attributes, rankingFilterState.attributes, isEqual),
+    );
+    const measureNotChanged = isEqual(rankingFilter.measure, rankingFilterState.measure);
+
+    return operatorNotChanged && valueNotChanged && attributesNotChanged && measureNotChanged;
+};
+
+interface IRankingFilterDropdownBodyComponentOwnProps {
+    measureItems: IMeasureDropdownItem[];
+    attributeItems: IAttributeDropdownItem[];
+    filter: IRankingFilter;
+    onApply: (filter: IRankingFilter) => void;
+    onCancel: () => void;
+    onDropDownItemMouseOver?: (localIdentifier: string) => void;
+    onDropDownItemMouseOut?: () => void;
+}
+
+type RankingFilterDropdownBodyComponentProps = IRankingFilterDropdownBodyComponentOwnProps &
+    WrappedComponentProps;
+
+const RankingFilterDropdownBodyComponent: React.FC<RankingFilterDropdownBodyComponentProps> = ({
+    measureItems,
+    attributeItems,
+    filter,
+    onApply,
+    onCancel,
+    onDropDownItemMouseOver,
+    onDropDownItemMouseOut,
+    intl,
+}) => {
+    const rankingFilter = filter.rankingFilter;
+    const [value, setValue] = useState(rankingFilter.value);
+    const [operator, setOperator] = useState(rankingFilter.operator);
+    const [measureIdentifier, setMeasureIdentifier] = useState(
+        (rankingFilter.measure as LocalIdRef).localIdentifier,
+    );
+    const [attributeIdentifier, setAttributeIdentifier] = useState(
+        (rankingFilter.attributes?.[0] as LocalIdRef)?.localIdentifier,
+    );
```

Comment:
Check functions `measureLocalId` and `attributeLocalId`

Reply:
Our measure and attribute would need to have different interface in order to use it :(